README.md

```diff
@@ -126,3 +130,45 @@ logging best-practices
     - "info"
 - `title` (string) - the event that occurred
 - `data` (object) - other parameters describing the event
+
+### Middleware
+
+Kayvee includes logging middleware, compatible with expressJS.
+
+The middleware can be added most simply via
+
+```js
+kayveeMiddleware = require('kayvee/middleware')
+
+var app = express()
+app.use(kayveeMiddleware())
+```
+
+It also supports user configuration via an `options` object.
+It prints the values of the headers or the results of the handlers.
+If a value is `undefined`, the key will not be printed.
```

Comment:
Followup from offline convo with @agclever 

1. add this priority ordering to docs, so users are aware of it.
2. make the `reserved` / base fields configureable... in case the user wants to change them, or remove one so  that a future handler can write to that field. (rare use case for us, but makes things more flexible)
3. don't name things with `clever` just call it `base` or something more generic